src/include/platform/CHIPDeviceConfig.h

```diff
@@ -1044,3 +1044,75 @@
 #ifndef CHIP_DEVICE_CONFIG_FIRMWARE_BUILD_TIME
 #define CHIP_DEVICE_CONFIG_FIRMWARE_BUILD_TIME __TIME__
 #endif
+
+// -------------------- Device DNS-SD Advertising Configuration --------------------
+
+/**
+ * CHIP_DEVICE_CONFIG_DEVICE_TYPE
+ *
+ * Type of device using the CHIP Device Type Identifier.
+ */
+#ifndef CHIP_DEVICE_CONFIG_DEVICE_TYPE
+#define CHIP_DEVICE_CONFIG_DEVICE_TYPE 81 // 81 = 0x0051 Smart Plug
```

Comment:
Feels like this could be a test type? Just worried about this leaking through and causing confusion